drivers/display/display_ili9xxx.c

```diff
@@ -314,6 +317,11 @@ static int ili9xxx_configure(const struct device *dev)
 		return r;
 	}
 
+	r = config->regs_init_fn(dev);
+	if (r < 0) {
+		return r;
+	}
+
```

Comment:
I moved `regs_init_fn` call here because applying display configuration before init doesn't result in the desired effect.
Would be great if someone has one of the other display controllers `9341` or `9488` to verify that this change didn't affect them.

Reply:
it looks like something was up with my Env, i reverted this change.